app/src/main/java/com/example/playlistmaker/search/domain/impl/SearchTracksInteractorImpl.kt

```diff
@@ -5,11 +5,11 @@ import com.example.playlistmaker.search.domain.api.TracksRepository
 import com.example.playlistmaker.search.domain.consumer.Consumer
 import com.example.playlistmaker.search.domain.models.Track
 
-class SearchTracksInteractorImpl(private val repositiry: TracksRepository) :
+class SearchTracksInteractorImpl(private val trackRepositiry: TracksRepository) :
     SearchTracksInteractor {
 
     override fun getMusic(term: String, consumer: Consumer<List<Track>>) {
```

Comment:
🍏 Лучше называть аргументы так же, как они называются в интерфейсе или суперклассе, чтобы не возникало конфликтов:
```suggestion
    override fun getMusic(expression: String, consumer: Consumer<List<Track>>) {
```